src/config/settings.ts

```diff
@@ -115,7 +116,13 @@ export function init(availableVersions: readonly string[]): Settings {
     if (err instanceof InvalidStoredSettings) {
       throw (err);
     }
-    // Create default settings
+    // Use default settings
+    if (err.code === 'ENOENT' && os.platform() === 'darwin') {
+      const availMemoryInGB = os.totalmem() / 2 ** 30;
```

Comment:
```suggestion
      const totalMemoryInGB = os.totalmem() / 2 ** 30;
```
Because not all memory is available… :)